docs/pages/x/api/charts/default-charts-axis-tooltip-content.js

```diff
@@ -10,9 +10,9 @@ export default function Page(props) {
 
 Page.getInitialProps = () => {
   const req = require.context(
-    'docsx/translations/api-docs/charts',
+    'docsx/translations/api-docs/charts/default-charts-axis-tooltip-content',
```

Comment:
@alexfauquette Should we remove old translations now?

<img width="347" src="https://github.com/mui/mui-x/assets/13808724/c01d13fb-4d35-4fb5-8352-5e1868a84f7a">

Reply:
Looks much better now 👍🏻

<img width="210" src="https://github.com/mui/mui-x/assets/13808724/9bc13389-9545-4805-871d-ac81ab391414">